stix_shifter/stix_translation/src/modules/aws_cloud_watch_logs/json/from_stix_guardduty_map.json

```diff
@@ -0,0 +1,71 @@
+{
+  "log-type": "guardduty",
+  "ipv4-addr": {
+    "fields": {
+      "value": ["eth0_private_ip", "eth1_private_ip", "public_ip", "remote_ip"],
+      "x_com_aws_interface_id": ["interface_id"],
+      "x_com_aws_remote_city_name": ["city_name"],
+      "x_com_aws_remote_country_name": ["country_name"]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": ["ipv6_address"],
+      "x_com_aws_interface_id": ["interface_id"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["local_port"],
+      "dst_port": ["remote_port"],
+      "src_ref.value": ["eth0_private_ip"],
+      "dst_ref.value": ["remote_ip"],
+      "protocols[*]": ["protocol"]
+    }
+  },
+  "domain-name":{
+    "fields": {
+      "value": ["eth0_private_dns_name","eth1_private_dns_name", "public_dns_name", "dns_domain"]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": ["principal_id"],
+      "account_login": ["user_name"]
+    }
+  },
+  "x_com_aws": {
```

Comment:
Custom STIX objects should use dashes, in the form of `x-com-aws`. Underscores are used for properties on the STIX object, as you've correctly done with `x_com_aws_interface_id` for the `ipv4-addr` object.

Reply:
It will be updated

Reply:
done